scripts/bash_lib.sh

```diff
@@ -468,10 +479,7 @@ check_mariadb_server_and_create_structures() {
   sudo mariadb -e "CREATE PROCEDURE db.p() SELECT * FROM db.v_merge"
   sudo mariadb -e "CREATE FUNCTION db.f() RETURNS INT DETERMINISTIC RETURN 1"
   if [[ $test_mode == "columnstore" ]]; then
```

Comment:
A failure here is handled in the parent script i.e. deb/rpm - upgrade?

Reply:
right, set -e/+e in this function incorrect.